### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by andgarzonmal**
this is my capstone project hope you like it

- [ ] please take a look at my video describing my project https://www.loom.com/share/312196bfcdb8414682109d289f7aa6d2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+          <img class="image-sponsore" src="./images/lumen.png" alt="lumen">
+        </li>
+      </ul>
+    </article>
+  </section>
+
+  <footer class="footer">
+    <img class="footer-img" src="./images/music3.jpg" alt="logo2">
+    <p>All rights reserved <br> this web page is made only whit academic purposes</p>
+  </footer>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly remove the sponsors section from the About page to match the original design.
- [ ] Please change the footer background color on the About page to match the Behance design.

| Your design | Behance design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @andgarzonmal, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## To Highlight
- Your design looks really nice, good job 👏🏻 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Your readme looks professional ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @andgarzonmal 😃, 

Great work with the required changes. 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (andgarzonmal) [reply index.html]: thanks for your suhgeestions 
i just have made the required changes 

image

**Pull request merged**